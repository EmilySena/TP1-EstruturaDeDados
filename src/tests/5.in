25
458
636
102
488
183
826
824
626
240
287
268
532
701
501
547
389
124
757
344
745
138
242
680
692
452
0
0
0
0
0
0
0
0
0
0
-2
-2
242
452
-1
-1
-2
0
-2
124
-3
344
-3
757
-3
-1
-3
-1
-3
-2