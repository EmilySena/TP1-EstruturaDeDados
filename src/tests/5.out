nave 452 em combate
nave 692 em combate
nave 680 em combate
nave 242 em combate
nave 138 em combate
nave 745 em combate
nave 344 em combate
nave 757 em combate
nave 124 em combate
nave 389 em combate
547
501
701
532
268
287
240
626
824
826
183
488
102
636
458
547
501
701
532
268
287
240
626
824
826
183
488
102
636
458
nave 242 avariada
nave 452 avariada
nave 242 consertada
nave 452 consertada
452
242
547
501
701
532
268
287
240
626
824
826
183
488
102
636
458
nave 452 em combate
242
547
501
701
532
268
287
240
626
824
826
183
488
102
636
458
nave 124 avariada
124
nave 344 avariada
124
344
nave 757 avariada
124
344
757
nave 124 consertada
344
757
nave 344 consertada
757
344
124
242
547
501
701
532
268
287
240
626
824
826
183
488
102
636
458
